### PR TITLE
DOCSP-45350-mongosync-limitations-always-in-effect

### PR DESCRIPTION
**Pull request by amalhotra-mdb**
## DESCRIPTION
Add an `important` admonition that clarifies that the limitations are still in effect even after pausing/stopping mongosync.

## STAGING
https://deploy-preview-505--docs-cluster-to-cluster-sync.netlify.app/reference/limitations/#limitations

## JIRA
https://jira.mongodb.org/browse/DOCSP-45350

## SELF-REVIEW CHECKLIST

- [ ] Does each file have 3-5 taxonomy facet tags?
  See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files) 
- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -64,6 +64,11 @@ General Limitations
 - You can't sync a collection with any documents that have an empty timestamp,
   such as ``Timestamp(0,0)`` on pre-6.0 source clusters. 
 
+.. important::
+
+   ``mongosync`` enforces the documented limitations, regardless of whether
```
Comment (kennethdyer) [inline source/reference/limitations.txt:69]: Succinctness:

```
``mongosync`` enforces the limitations...
```

The "documented limitations" implies undocumented limitations.

**Review by kennethdyer — APPROVED** [review]
Thanks @amalhotra-mdb, this LGTM with one suggestion.

File: source/reference/limitations.txt (hunk 1 above)
Comment (amalhotra-mdb) [reply source/reference/limitations.txt]: Ah that makes sense. Thank you for this fix!

**Review by rohshar — CHANGES_REQUESTED** [review]
Why is this not listed in the top level warning? I think it would be better there. 

Also, mongosync doesn't enforce all limitations, we can't say that. Just that they have to adhere to the limitaitons for the full duration of the migration, including when the migration is paused / or stopped if it may be resumed

**Comment by amalhotra-mdb** [comment]
> Why is this not listed in the top level warning? I think it would be better there.
> 
> Also, mongosync doesn't enforce all limitations, we can't say that. Just that they have to adhere to the limitaitons for the full duration of the migration, including when the migration is paused / or stopped if it may be resumed

Thanks for your review, @rohshar. I agree, this information should b e in the top level warning – I went ahead and added it there. Thanks for your content changes – all changes made! Could you take a second look and see if it looks okay to you?

File: source/reference/limitations.txt [hunk 2]
```diff
@@ -21,6 +21,10 @@ Limitations
    these limitations could lead to undefined behavior on the destination
    cluster.
 
+   You must adhere to these limitations for the full duration of the the
+   migration, including when the migration is paused or stopped and will be
```
Comment (rohshar) [inline source/reference/limitations.txt:25]:
```suggestion
   migration, including when the migration is paused or stopped if it will be
```

**Review by rohshar — APPROVED** [review]
one nit, then LGTM

**Pull request merged**